sdk/monitor/monitor-query/src/models/publicLogsModels.ts

```diff
@@ -106,31 +121,36 @@ export interface QueryBatch {
 /** Results for a batch query. */
 export interface LogsQueryBatchResult {
   /** An array of responses corresponding to each individual request in a batch. */
-  results?: {
-    id?: string;
-    status?: number;
-    /** The list of tables, columns and rows. */
-    // (hoisted up from `LogQueryResult`)
+  results: {
```

Comment:
Is this just `LogsQueryResult[]` or `Array<LogsQueryResult>`? Maybe we can reuse the type instead of duplicating if so

Reply:
https://github.com/Azure/azure-sdk-for-js/issues/17545